src/tokens.ts

```diff
@@ -556,6 +563,14 @@ export namespace Git {
        * Reference data
        */
       reference: IContent<T>;
+      /**
+       * Optional base data, used only during merge conflicts
+       */
+      base?: IContent<T>;
+      /**
+       * Helper to check if the file is conflicted.
+       */
+      isConflict: boolean;
```

Comment:
To be coherent this should be undefined if `base` is undefined.

```suggestion
      hasConflict?: boolean;
```